src/Sql/dbo/Stored Procedures/CollectionUser_UpdateUsers.sql

```diff
@@ -30,18 +29,21 @@ BEGIN
         AND (
             [Target].[ReadOnly] != [Source].[ReadOnly]
             OR [Target].[HidePasswords] != [Source].[HidePasswords]
-            OR [Target].[Manage] != [Source].[Manage]
         )
 
-    -- Insert
-    INSERT INTO
-        [dbo].[CollectionUser]
+    -- Insert (now with column list)
+    INSERT INTO [dbo].[CollectionUser]
+    (
+        [CollectionId],
+        [OrganizationUserId],
+        [ReadOnly],
+        [HidePasswords]
+    )
```

Comment:
Assuming this is correct - a slightly more explanatory comment might be helpful

```sql
-- Insert (with column list because a value for Manage is not being provided)
```

Reply:
Yep, that's correct and I'll update the comment to be more clear.